src/lib/multiaddr.ts

```diff
@@ -22,11 +22,14 @@ import {
   testNet,
   Node,
 } from "@fluencelabs/fluence-network-environment";
-import shuffle from "lodash/shuffle";
+import lodash from "lodash";
```

Comment:
I believe there is a special package for lodash with ESM, https://www.npmjs.com/package/lodash-es
Have you tried using it instead of `lodash`?

Reply:
no I did not. I will. Thanks